PublicationsRouter/v2/api/API.md

```diff
@@ -158,7 +176,7 @@ On **validation failure** the system will respond with the following:
 
 On **validation success** the system will respond with 204 (No Content) and no response body.
 
-### Notification Endpoint
+## Notification Endpoint
 
 The Notification API takes an identical request to the Validation API, so that you can develop against the Validation API and then switch seamlessly over to live notifications.  The only difference will be in the response body.
```

Comment:
Perhaps this should be changed to:
```
The Notification API endpoints takes an identical request (header and body) to the Validation API endpoints, so that you can develop against the Validation API and then switch seamlessly over to live notifications.  However, there will be a difference in the response body that is received.
```